api/leap.py

```diff
@@ -133,3 +133,55 @@ def _get_compute_stub(self):
     def _create_request_obj(self):
         request = pb.computation_msgs_pb2.ComputeRequest()
         return request
+
```

Comment:
Inherit from the DistributedLeap class. Some of the attributes in the class are the same and the get_compute_stub function is the same, so you can reuse that. The others you can override as you wish.